docs/start/multinode.rst

```diff
@@ -71,6 +71,119 @@ Slurm
 -----
 TBD
 
+dstack
+------
+If you want to run multi-node training with `dstack <https://dstack.ai/>`_ on NVIDIA Cluster, you need to follow the following steps.
+
+1. Prerequisite
+
+   Once dstack is `installed <https://dstack.ai/docs/installation/>`_, go ahead clone the repo, and run dstack init. 
+
+   .. code-block:: bash
+
+       $ git clone https://github.com/dstackai/dstack
```

Comment:
Avoid suggesting cloning our example. Maybe, suggest creating a new folder (like in our quickstart)?